packages/opencensus-core/src/trace/model/no-record/no-record-root-span.ts

```diff
@@ -31,6 +29,8 @@ export class NoRecordRootSpan extends NoRecordSpan {
    * parent was likely started on another machine.
    */
   private parentSpanIdLocal: string;
+  /** A tracer object */
+  tracer: types.TracerBase;
```

Comment:
Could this be `readonly`? (Applies to other uses below too)

Reply:
I changed it